concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1489,6 +1489,27 @@ jobs:
                 # when we roll out the London region.
                 cf create-route docs "${APPS_DNS_ZONE_NAME}" --hostname london
 
+      - task: configure-wildcard-routes
```

Comment:
We already have a place for this: https://github.com/alphagov/paas-cf/blob/894afdd5ac60220448ba12635f082f0e66f51845/concourse/pipelines/create-cloudfoundry.yml#L1481-L1486

Reply:
Fixed. I put it in `govuk-paas/docs` now